packages/block-library/src/comment-content/index.php

```diff
@@ -28,6 +30,21 @@ function render_block_core_comment_content( $attributes, $content, $block ) {
 		return '';
 	}
 
+	$moderation_note = '';
+	if ( '0' === $comment->comment_approved ) {
+		$commenter = wp_get_current_commenter();
+
+		if ( $commenter['comment_author_email'] ) {
+			$moderation_note = __( 'Your comment is awaiting moderation.' );
+		} else {
+			$moderation_note = __( 'Your comment is awaiting moderation. This is a preview; your comment will be visible after it has been approved.' );
+		}
+		$moderation_note = '<em class="comment-awaiting-moderation">' . $moderation_note . '</em>';
```

Comment:
```suggestion
		$moderation_note = '<p><em class="comment-awaiting-moderation">' . $moderation_note . '</em></p>';
```

Associated changes will be needed in the tests too.

Reply:
I've added this change. Now it even looks a bit better. 😄 

![Screenshot 2022-04-28 at 13 41 29](https://user-images.githubusercontent.com/6917969/165744674-cf17ceb1-167a-4cc5-807d-52de0b887245.png)